week1/Afshin Week1.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # ex1
+
+name = input ("Enter your name :")
+surname = input ("Enter your surname :")
+string_print = f"Your name is {name} and your family is {surname }."
+print (string_print)
+
+
+# ex2
+i = 1
+x = ""
+while i < 6:
+    film = input("Enter name of your favorite film :")
+    x = film + ',' + x + '  '
+    print(x)
+    i += 1
+
+
+
+
+
+# ex3
+
+def printfirstlast():
+  color_list = ["Red","Green","White" ,"Black"]
+  print(color_list[0])
+  print(color_list[-1])
+
+
+
+
+printfirstlast()
+
+
+# ex4
+
+mydictis = {
+  0: 10,
+  1: 20
+}
+
+
+mydictis[2]=30
+
+
+mydictis
+
+
+# ex5
+
+def body():
+    weight = input ("Enter weight (kg) :")
+    height = input ("Enter weight (m) :")
+    bmi = float(weight) / float(height)**2
+    print(bmi)
+
+
+body()
+
+
+
+
+
+
+# ex6
+
+
+i = 1
+while i == 1:
+  inputNumber = input ("Your guess :")
+  print(inputNumber)
+  if inputNumber == "6":
```

Comment:
it would be better to cast the `inputNumber` to an integer, and then don't do the comparison in the domain of strings, but in the domain of integers.
something like `inputNumber == "6"` is really confusing, because:
first, `inputNumber` actually holds a string, and `"6"` is a number as a string.